ingest/ingest_pipeline.py

```diff
@@ -405,7 +405,8 @@ def ingest_expression(self) -> int:
     # @my_debug_logger()
     def ingest_cell_metadata(self):
         """Ingests cell metadata files into Firestore."""
-        if self.cell_metadata.validate_format():
+        if self.cell_metadata.validate():
+            pass
```

Comment:
This `pass` seems extraneous -- can it be removed?

```suggestion
```

Reply:
0f2b689